File: Programming-languages-and-laboratory/Python/Exercises/40_compito_python_20220620/Noleggio.py
class Noleggio:
    def __init__(self, targa, giorni, costo):
        self._targa = targa
        self._giorni = giorni
        self._costo = costo

    def getTarga(self):
        return self._targa

    def getCosto(self):
        return self._giorni * self._costo

    def __str__(self):
        return "(" + str(self._targa) + "," + str(self._giorni) + ")"

File: Programming-languages-and-laboratory/Java/Exercises/40_compito_java_2020_06_22/Noleggio.java
public class Noleggio {
    protected String veicolo;
    protected int giorni;
    protected double costo;

    public Noleggio(String veicolo, int giorni, double costo) {
        this.veicolo = veicolo;
        this.giorni = giorni;
        this.costo = costo;
    }

    public double getTot() {
        return giorni * costo;
    }

    public String toString() {
        return "(" + veicolo + "," + giorni + ")";
    }
}
